.github/CODEOWNERS

```
@@ -42,7 +42,7 @@
 /go/ @zeroshade
 /java/ @lidavidm
 /js/ @domoritz @trxcllnt
-/matlab/ @assignUser
+/matlab/ @assignUser @kevingurney
```

```suggestion
/matlab/ @kevingurney
```
As I only added myself to catch any stray matlab PRs that need you tagged on I can now be removed, congrats again!

---

Thank you, @assignUser! :)